### PR TITLE
Refactor Python examples with a new "util.py" file

### PR DESCRIPTION
**Pull request by ManifoldFR**
This PR refactors a bunch of Python examples to use a function from the same `util.py` file

This helps with my draft PR #345 but should have value in itself

### CODE REVIEW

File: examples/python/util.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import scipy.sparse as spa
+
+
+def generate_mixed_qp(n, seed=1, reg=-2.0, dens1=0.075):
```
Comment (fabinsch) [inline examples/python/util.py:5]: `reg` is not used. we should either remove it, as it is not used by any of our examples for the dense backend, or give it a meaning like [here](https://github.com/Simple-Robotics/proxsuite/blob/main/test/src/sparse_qp_wrapper.py#L34) for future usage.

Comment (ManifoldFR) [reply examples/python/util.py:5]: You're right, it was already in the code but never used.

Comment (ManifoldFR) [reply examples/python/util.py]: done

**Review by fabinsch — APPROVED** [review]

File: examples/python/util.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import scipy.sparse as spa
+
+
+def generate_mixed_qp(n, sparse=False, seed=1, reg=1e-2, dens1=0.075):
+    # A function for generating sparse random convex qps in dense format
```
Comment (fabinsch) [inline examples/python/util.py:6]:
```suggestion
    # A function for generating sparse random convex qps
```

**Review by fabinsch — DISMISSED** [review]

**Review by jcarpent — APPROVED** [review]

**Pull request merged**